src/test/java/kanbanflow/steps/KanbanStep.java

```diff
@@ -155,10 +155,32 @@ public void theUserCloseModalWindows() {
 
     /**
      * Method to let description in the board.
```

Comment:
the documentation must be written in third person

Reply:
That is in third person.